packages/js/internal-js-tests/jest-preset.js

```diff
@@ -51,6 +51,9 @@ module.exports = {
 		'**/test/*.[jt]s?(x)',
 		'**/?(*.)test.[jt]s?(x)',
 	],
+	testPathIgnorePatterns: [
```

Comment:
`global.d.ts` was being run by jest since I've introduced one in test folder in this PR